### PR TITLE
✨Make Linter More Interactive

### PR DESCRIPTION
**Pull request by Vyperus**
**Changes:**

1. The icon of the linter button changes dependent on the lint results.
2. The start-/save-/deploy-button now got correct tooltips.

**Issues:**

Part of: #1405 

PR: #PullRequest

## How can others test the changes?

1. Open a diagram.
2. Delete the start event.
2.1 Notice that the linter button now shows a cross.
2.2. Notice that the deploy-/start-/save-buttons now have other tooltips.

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

### CODE REVIEW

File: src/modules/navbar/navbar.ts [hunk 1]
```diff
@@ -259,6 +259,38 @@ export class NavBar {
     this._eventAggregator.publish(environment.events.diagramDetail.uploadProcess);
   }
 
+  public get diagramUploadButtonTitle(): string {
+    if (this.disableDiagramUploadButton) {
+      return 'This process is already deployed to the remote ProcessEngine.';
+    }
+
+    if (this.validationError) {
+      return 'There was a problem with this diagram. Please check the linter for more Information.';
```
Comment (alexanderkasten) [inline src/modules/navbar/navbar.ts]:
```suggestion
      return 'There was a problem with this diagram. Please check the linter for more information.';
```

File: src/modules/navbar/navbar.ts [hunk 2]
```diff
@@ -259,6 +259,38 @@ export class NavBar {
...
+    return 'Deploy to ProcessEngine';
+  }
+
+  public get startButtonTitle(): string {
+    if (this.disableStartButton) {
+      return 'Before starting the process please deploy it to the ProcessEngine.';
+    }
+
+    if (this.validationError) {
+      return 'There was a problem with this diagram. Please check the linter for more Information.';
```
Comment (alexanderkasten) [inline src/modules/navbar/navbar.ts]:
```suggestion
      return 'There was a problem with this diagram. Please check the linter for more information.';
```

File: src/modules/navbar/navbar.ts [hunk 3]
```diff
@@ -259,6 +259,38 @@ export class NavBar {
...
+    }
+
+    return 'Start Process';
+  }
+
+  public get saveButtonTitle(): string {
+    if (this.validationError && this.savingTargetIsRemoteSolution) {
+      return 'There was a problem with this diagram. Please check the linter for more Information.';
```
Comment (alexanderkasten) [inline src/modules/navbar/navbar.ts]:
```suggestion
      return 'There was a problem with this diagram. Please check the linter for more information.';
```

File: src/modules/navbar/navbar.ts (hunk 1 above)
Comment (Vyperus) [reply src/modules/navbar/navbar.ts]: Nope, I guess you are thinking about the german "Information" here.
https://dict.leo.org/german-english/information

File: src/modules/navbar/navbar.ts (hunk 2 above)
Comment (Vyperus) [reply src/modules/navbar/navbar.ts]: See above

File: src/modules/navbar/navbar.ts (hunk 3 above)
Comment (Vyperus) [reply src/modules/navbar/navbar.ts]: See above.

File: src/modules/navbar/navbar.ts (hunk 1 above)
Comment (alexanderkasten) [reply src/modules/navbar/navbar.ts]: Nope, https://www.lern-online.net/englisch/schreiben/rechtschreibung/gross-und-kleinschreibung/#schluss-anredeformen-brief

Comment (Vyperus) [reply src/modules/navbar/navbar.ts]: Oh sorry I misread the suggestion entirely, my mistake.

File: src/modules/navbar/navbar.ts [hunk 4]
```diff
@@ -259,6 +259,38 @@ export class NavBar {
...
+    if (this.validationError) {
+      return 'There was a problem with this diagram. Please check the linter for more information.';
+    }
+
+    return 'Deploy to ProcessEngine';
+  }
+
+  public get startButtonTitle(): string {
+    if (this.disableStartButton) {
+      return 'Before starting the process please deploy it to the ProcessEngine.';
```
Comment (ElRaptorus) [inline src/modules/navbar/navbar.ts]:
```suggestion
      return 'Please deploy the process to a ProcessEngine before starting it.';
```

**Pull request merged**